nutanix/data_source_nutanix_project.go

```diff
@@ -1,353 +1,353 @@
 package nutanix
 
-import (
-	"fmt"
+// import (
```

Comment:
I'd go for uncommenting and add .future extension to avoid lint issues.

Reply:
let me change that thanks